debian/usr.bin.snap-confine

```diff
@@ -112,6 +114,15 @@
     mount options=(rw bind) /snap/ubuntu-core/*/libx32/ -> /libx32/,
     mount options=(rw bind) /snap/ubuntu-core/*/lib64/ -> /lib64/,
     mount options=(rw bind) /snap/ubuntu-core/*/usr/ -> /usr/,
+    mount options=(rw bind) /snap/ubuntu-core/*/etc/alternatives/ -> /etc/alternatives/,
+
+    # This is a somewhat of a blank check for mount profiles.  Mount profiles
+    # are written by the trusted snapd to a root-writable location and
+    # processed by snap-confine. Regardless of configuration options and
+    # runtime factors they are processed after pivot_root (on classic systems)
+    # so snap.rootfs is not a factor.
+    mount options=(rw bind) /** -> /**,
```

Comment:
The comment as it stands is unfortunate. However, I'd like to challenge that assessment. :)

Doesn't snapd control what the mount points are via interfaces? As such, we are still in control of the mountpoints within the snapcore project.

Even if not, aren't the mount for things to be mounted inside the snap? As such, we should be able to limit the target directory at the very least. For example, it would be exceedingly unfortunate if we mounted /var as rw in the snap writable area since then the snap has control to modify /var/lib/snapd/....

With that in mind, I'm now looking at these mount rules and wondering why many of them are 'rw'-- eg, everything under 'when --enable-rootfs-is-core-snap is NOT used' and nvidia.